recipes/catalystcoop.arelle-mirror/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "catalystcoop.arelle-mirror" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/catalystcoop.arelle_mirror-{{ version }}.tar.gz
+  sha256: 161a6a829cb830683b3136cff8dac8797f2924189d07bafff57d87c775708247
+
+build:
+  entry_points:
+    - arelle=arelle.CntlrCmdLine:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
```

Comment:
Looks like this can be noarch

Reply:
I've updated to noarch. The build is failing on Windows, but we don't plan on supporting windows, is this ok?